docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1463,283 +1463,46 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0190I
-
-  Type requested = %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0191I
-
-  Returning true if type matches, type=%s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0192I
-
-  Returning true because type is application
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0193I
-
-  Returning false because type did not match
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0194I
-
-  Callservice: Data received
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0195I
-
-  Callservice: Service call completed.
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0196I
-
-  Callservice: Issuing request to service: %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0197I
-
-  User=%s: %s service called: %s, %s %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0198I
-
-  User=%s: Service called: %s::%s, %s %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0199I
-
-  %s: service %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0200I
-
-  %s: %s %s handled by %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0201I
-
-  About to call myProxy
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0202I
-
-  After myProxy call
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0203I
-
-  Referrer proxying miss. Resource not found, sending 404 because referrer (%s) didn't match an existing proxy service
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0204I
-
-  Referrer proxying miss. Resource not found, sending 404 because referrer (%s) didn't match a plugin pattern
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
 ### ZWED0205I
 
-  User=%s (%s): User logout
+  User=_user_ (_pluginId_): User logout
 
   **Reason:**
 
-  TODO
+  This message prints when the _user_ logs out of the Zowe Desktop. Logout is being handled by the _pluginId_ plugin.
 
   **Action:**
 
-  TODO
-
-
-
-### ZWED0206I
-
-  Writing: %s, Size=%s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0207I
-
-  Wrote: %s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
+  If logout was intentional, message can be safely ignored. If logout was intentional, keep in mind the Desktop logs out after inactivity. Incorrect logout behavior can be troubleshooted with
```

Comment:
Is it supposed to be like: If logout was **unintentional**, keep in mind the Desktop logs out after inactivity?